File-Formats/Presentation/Working-with-find-and-replace.md

```diff
@@ -299,6 +299,80 @@ End Using
 
 You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/PowerPoint-Examples/tree/master/Find-and-Replace/Find-first-occurance).
 
+
+## Find and replace using Regex pattern 
+
+You can find all occurrences of a pattern of text in a PowerPoint presentation using the FindAll method and replace it with other text.
+ 
+The following code example illustrates how to find all occurrences of a pattern of text and replace it with other text in a PowerPoint presentation using Regex pattern.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C# [Cross-platform]" %}
+
+//Load or open an PowerPoint Presentation.
+using (FileStream inputStream = new FileStream ("Sample.pptx", FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
+{
+    //Open an existing PowerPoint presentation.
+    using (IPresentation pptxDoc = Presentation.Open(inputStream))
+    {
+        //Find all the occurrences of a particular pattern text in the PowerPoint presentation using Regex.
```

Comment:
@Venkateshmuruganandam - Please modify the XML comments as below,
`//Find all the occurrences of given pattern of text in the PowerPoint presentation using Regex.`